diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -103,17 +105,25 @@ enum class Warnings(private val canBeAutoCorrected: Boolean, private val warn: S
                    isFixMode: Boolean,
                    freeText: String,
                    offset: Int,
+                   node: ASTNode?,
```

Comment:
Why is `node` nullable? I think that if we call `warn` or `warnAndFix` we should have some node available.